<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="Vertical">
       <item>
        <widget class="QPushButton" name="New">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Open_2">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Import">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>431</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Nothingpressed"/>
      <widget class="QWidget" name="Openstacks">
       <widget class="QPushButton" name="Next">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>460</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>174</x>
          <y>250</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Case Name      :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>180</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Exit_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>460</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="NewStack">
       <widget class="QPushButton" name="Next2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>460</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Case Name      :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Working Directory      :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Exit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>460</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QStackedWidget" name="stackedWidget_3">
      <property name="geometry">
       <rect>
        <x>102</x>
        <y>19</y>
        <width>632</width>
        <height>458</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_15"/>
      <widget class="QWidget" name="page_16">
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>77</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Shapelayout_7">
         <item>
          <widget class="QPushButton" name="Rectangle_7">
           <property name="text">
            <string>Rectangle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Circle_7">
           <property name="text">
            <string>Circle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Line_7">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Polygon_7">
           <property name="text">
            <string>Polygon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Add_7">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Remove_7">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>77</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="MainLayout_7">
       <item>
        <widget class="QPushButton" name="Drawing_7">
         <property name="text">
          <string>Drawing</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_25">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Boundaries_7">
         <property name="text">
          <string>Boundaries</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_26">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Mesh_7">
         <property name="text">
          <string>Mesh</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_27">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Materials_7">
         <property name="text">
          <string>Materials</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_28">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Run_7">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menudBM_Launcher">
    <property name="title">
     <string>dBM Launcher</string>
    </property>
   </widget>
   <addaction name="menudBM_Launcher"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
